### PR TITLE
Coach tools plugin boilerplating

### PR DESCRIPTION
**Pull request by DXCanas**
## Summary

Setting up a skeleton for the plug to be build on, includes core-base template and nav bar.

## Screenshots (if appropriate)
image
image

### CODE REVIEW

**Comment by DXCanas** [comment]
[add a new 'coach-tools' placeholder plugin](https://trello.com/c/EJv269jQ/622-add-a-new-coach-tools-placeholder-plugin)

File: kolibri/plugins/coach_tools/assets/src/app.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const coreActions = require('kolibri/coreVue/vuex/actions');
+
+const rootvue = require('./vue');
+const actions = require('./actions');
+const store = require('./state/store');
+
+class CoachToolsModule extends KolibriModule {
+  ready() {
+    router.on(
+      'HOME',
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/app.js]: `HOME` and `SCRATCHPAD` should be defined in _coach_tools/assets/src/state/constants.js_ under a `PageNames` object.

(See e.g. Learn's constants file for an example.)

File: kolibri/plugins/coach_tools/assets/src/state/store.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const Vuex = require('vuex');
+const coreStore = require('kolibri/coreVue/vuex/store');
+// const constants = require('./constants');
+
+const initialState = {
+  pageName: 'HOME',
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/state/store.js:6]: and here we'll grab the page name constant from `constants`

File: kolibri/plugins/coach_tools/assets/src/vue/index.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<template>
+
+  <core-base>
+    <main-nav slot="nav"></main-nav>
+    <div v-if="!currentPage" slot="content"> <h1> Who likes short shorts? </h1> </div>
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/vue/index.vue:5]: we like short shorts

File: kolibri/plugins/coach_tools/kolibri_plugin.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from kolibri.core.webpack import hooks as webpack_hooks
+from kolibri.plugins.base import KolibriPluginBase
+from . import hooks, urls
+
+
+class CoachTools(KolibriPluginBase):
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/kolibri_plugin.py:7]: We should add this plugin to the 'default plugins' list so it doesn't need to be manually installed:

https://github.com/learningequality/kolibri/blob/master/kolibri/utils/conf.py#L47

**Comment by DXCanas** [comment]
@indirectlylit Made changes and rebased, should be read to merge. Let me know if you see any other glaring issues though!

**Comment by indirectlylit** [comment]
Hey @DXCanas - not sure what happened with your rebase, but there are a lot more commits in this PR than there should be

**Comment by DXCanas** [comment]
Yeah I'm not sure either @indirectlylit, I just did a `git rebase` to LE's develop branch to get rid of conflicts.just did it again and whatever reason all these changes are associated with my feature branch.

**Comment by DXCanas** [comment]
That looks right. lemme know!

**Pull request merged**